crazyflie/CMakeLists.txt

```diff
@@ -54,6 +55,7 @@ ament_target_dependencies(crazyflie_server
   sensor_msgs
   std_srvs
   crazyflie_interfaces
+  motion_capture_tracking_interfaces
```

Comment:
do not add in motion capture since we are not using or do not have motion capture module installed